Content.Server/GameObjects/Components/Buckle/BuckleComponent.cs

```diff
@@ -286,6 +289,16 @@ public override bool TryBuckle(IEntity user, IEntity to)
 
             Owner.EntityManager.EventBus.SubscribeEvent<MoveEvent>(EventSource.Local, this, MoveEvent);
 
+
+            PullableComponent? pullableComponent;
+            if (Owner.TryGetComponent(out pullableComponent))
```

Comment:
You can just do the following:
```suggestion
            if (Owner.TryGetComponent(out PullableComponent? pullableComponent))
```